diff --git a/node_modules/nuxt-primevue/dist/module.json b/node_modules/nuxt-primevue/dist/module.json
index 5d029a4..d830998 100644
--- a/node_modules/nuxt-primevue/dist/module.json
+++ b/node_modules/nuxt-primevue/dist/module.json
@@ -2,7 +2,7 @@
   "name": "nuxt-primevue",
   "configKey": "primevue",
   "compatibility": {
-    "nuxt": "^3.0.0"
+    "nuxt": "^4.0.0"
   },
   "version": "3.0.0"
 }
\ No newline at end of file
diff --git a/node_modules/nuxt-primevue/dist/module.mjs b/node_modules/nuxt-primevue/dist/module.mjs
index fd76a2e..6ef19a4 100644
--- a/node_modules/nuxt-primevue/dist/module.mjs
+++ b/node_modules/nuxt-primevue/dist/module.mjs
@@ -230,7 +230,7 @@ const module = defineNuxtModule({
     name: "nuxt-primevue",
     configKey: "primevue",
     compatibility: {
-      nuxt: "^3.0.0"
+      nuxt: "^4.0.0"
     }
   },
   defaults: {
diff --git a/node_modules/nuxt-primevue/dist/runtime/plugin.client.d.mts b/node_modules/nuxt-primevue/dist/runtime/plugin.client.d.mts
new file mode 100644
index 0000000..2abe959
--- /dev/null
+++ b/node_modules/nuxt-primevue/dist/runtime/plugin.client.d.mts
@@ -0,0 +1,3 @@
+import type { Plugin } from '#app';
+declare const _default: Plugin;
+export default _default;
diff --git a/node_modules/nuxt-primevue/dist/runtime/plugin.server.d.mts b/node_modules/nuxt-primevue/dist/runtime/plugin.server.d.mts
new file mode 100644
index 0000000..6e2f1b8
--- /dev/null
+++ b/node_modules/nuxt-primevue/dist/runtime/plugin.server.d.mts
@@ -0,0 +1,4 @@
+import type { NitroApp } from 'nitropack';
+type NitroAppPlugin = (nitro: NitroApp) => void;
+declare const _default: NitroAppPlugin;
+export default _default;
